### ISSUE TITLE
`run.py cover` broken

### ISSUE THREAD

**Issue by rogerbosman** [wannesm/adsphd#129]
The `run.py cover` script is broken: it fails during building of the generated tex.

Steps to reproduce:

1. Clone current (V1.6, 56920fa) repository
2. cd to directory
3. call `python3 run.py cover`

The script fails with
```
Command '['pdflatex', '-interaction=nonstopmode', '-synctex=1', '-shell-escape', 'cover']' returned non-zero exit status 1.
run.py: Running Latex failed (exitcode 1)
```

Running `latexmk ./cover.tex` halts with the following error at the end:

```
! Missing number, treated as zero.
<to be read again> 
!
l.82 \makefullcoverpage{\adsphdspinewidth}{}
```

**Comment by rogerbosman** [comment]
Note that fails in the same way in my personal repository with my thesis text. However, since it fails using the template in the repository, the problem should be in the script.

**Comment by wannesm** [comment]
Fixed in 9eb4a5f
